pulp_fedora31/Containerfile

```diff
@@ -1,20 +1,20 @@
 FROM pulp/pulp-ci:latest
 
-ARG PULPCORE_VERSION=3.4.1
-ARG PULP_FILE_VERSION=1.0.1
-ARG PULP_CONTAINER_VERSION=2.0.0b2
-ARG PULP_ANSIBLE_VERSION=0.2.0b13
-ARG PULP_RPM_VERSION=3.4.1
-ARG PULP_MAVEN_VERSION=0.1.0
+ARG PULPCORE_VERSION=""
+ARG PULP_FILE_VERSION=""
+ARG PULP_CONTAINER_VERSION=""
+ARG PULP_ANSIBLE_VERSION=""
+ARG PULP_RPM_VERSION=""
+ARG PULP_MAVEN_VERSION=""
```

Comment:
I think we can merge it if we declare default values

Reply:
So the point is not to have defaults but automatically use the latest versions from pypi.